### PR TITLE
update readme files with new build information

### PR DESCRIPTION
**Pull request by acoburn**
This amends https://github.com/fcrepo4-labs/fcrepo-camel-toolbox/pull/20 with updated README files

### CODE REVIEW

**Comment by acoburn** [comment]
Resolves https://jira.duraspace.org/browse/FCREPO-1527

File: fcrepo-camel-webapp/README.md [hunk 1]
```diff
@@ -8,10 +8,18 @@ for [Fedora4](http://fcrepo.org).
...
 
     MAVEN_OPTS="-Xmx1024m" mvn install -Pat
 
+To build this project with the solr and triplestore indexers, use
+
+    MAVEN_OPTS="-Xmx1024m" mvn install -Pis -Pit
+
+To build this project with all three applications, use
+
+    MAVEN_OPTS="-Xmx1024m" mvn install -Pis -Pit -Pat
```
Comment (awoods) [inline fcrepo-camel-webapp/README.md]: FYI, the following also works: `mvn install -Pis,it,at`

Comment (acoburn) [inline fcrepo-camel-webapp/README.md]: Ah, good to know.  That example would be nice to have in the readme file, too

Comment (awoods) [inline fcrepo-camel-webapp/README.md]: I will add the example directly.

**Pull request merged**